src/Serialization/Rings.jl

```diff
@@ -119,6 +119,9 @@ function load_internal_with_parent(s::DeserializerState,
                                    ::Type{<: MPolyElem},
                                    dict::Dict,
                                    parent_ring::MPolyRing)
+    # load parent in case serialzed parent needs to be checked against given parent
+    _, _ = load_unknown_type(s, dict[:parent])
```

Comment:
I don't quite understand: what problem does this solve? You ignore the result, so what does it do? Is this about caching somehow?

Reply:
Then ideally the comment should explain just that :-)